src/helm/proxy/models.py

```diff
@@ -666,7 +667,7 @@ def engine(self) -> str:
         display_name="ChatGPT",
         description="Sibling model to InstructGPT which interacts in a conversational way",
         # TODO: The max context length is unknown. Assume it's the same length as Davinci Instruct for now.
-        tags=[TEXT_MODEL_TAG, WIDER_CONTEXT_WINDOW_TAG, LIMITED_FUNCTIONALITY_TEXT_MODEL_TAG, GPT2_TOKENIZER_TAG],
+        tags=[TEXT_MODEL_TAG, WIDER_CONTEXT_WINDOW_TAG, GPT4_TOKENIZER_TAG, LIMITED_FUNCTIONALITY_TEXT_MODEL_TAG],
```

Comment:
Don't need to change `openai/chat-gpt`; we're planning to deprecate this soon in favor of gpt-3.5-turbo

Reply:
Reverted.